BazelExtensions/workspace.bzl

```diff
@@ -270,6 +285,7 @@ def new_pod_repository(name,
         target_name=name,
         url=url,
         podspec_url=podspec_url,
+        podspec_file=podspec_file,
```

Comment:
It might be possible to use the macro to provide a symmetrical API here to the user:
1) in the proceeding code, test if the specified `podspec_url`, has a scheme `http`
2) if HTTP load the URL, else pass the `string` into rule as a file somehow ( perhaps populate `podspec_file` ), so that it can resolve the path.

It seemed nice to make the PodSpec attribute mutually exclusive from the developer's perspective 👍

Reply:
Ah, that's a good idea! The macro works perfectly to allow me to resolve labels relative to the main workspace 😀 

I went ahead and pushed this change.